app/content/webviewPreload.js

```diff
@@ -156,7 +156,8 @@ ipc.on(messages.SET_AD_DIV_CANDIDATES, function (e, adDivCandidates, placeholder
 })
 
 function hasSelection (node) {
-  if (node && node.selectionStart !== undefined &&
+  if (node && /password|search|tel|text|url/.test(node.type) &&
```

Comment:
I'm not sure if we need this extra check, won't it prevent the copy menu from working on plain text? Is there a case that is broken that this fixes?